nebula-logger/core/main/logger-engine/classes/LoggerStackTrace.cls

```diff
@@ -166,84 +166,6 @@ public without sharing class LoggerStackTrace {
         this.Location = cleanedLocation;
     }
 
```

Comment:
nice! this is going to be a big performance improvement, the stacktrace parsing was still one of the big errors where bulk log entries were slowing down!

Reply:
Yeah, it took a while for me to come to terms with shifting this logic to JS, but I think in the end, getting rid of this Apex code is going to be great 🏎️

And I'm hoping to revisit 1 or 2 other parts of the remaining code (for parsing Apex stack traces) in the coming weeks - it's a bit out of scope for this PR, but I think I might be able to optimize some of that logic a bit to reduce CPU time some more.